### ISSUE TITLE
Dawn@Press

### ISSUE THREAD

**Issue by Debayan-Banerjee** [Debayan-Banerjee/Guzzle-Humdrum#14]
Press : "do you feel you underachieved Mrs Glenol(all laugh..lot)"
Dawn(straight) : "no i can't feel that way can i feeling is only yes no as you know(all burst..)"
Press : "well you started 14 years back the anniversary is a week to go of that journey your first ever that hasn't been repeated in the last 11 years atleast Mrs Glenol do you believe it will happen may be after 3 generations(all laugh..lot)"
Dawn(straight) : "well i can't comment on this cause they are discrete events as you understand but i can tell the only thing that i know is that you need logistics for that by yourself or someone else but mostly it must be desire and that's never about you it's about me there must be desire in me if i wanted to pull out i didn't have to tell anyone anything my bunch would've done that themselves as you subsequently saw(all laugh..lot)"
Press : "well you said many times you didn't know much at all about this profession before you began but over the course of this journey you might've surpassed your mentors in terms of capturing a whole region's attention that too in an extremely rare time for this profession when it's existence was questioned and they weren't there at all"
Dawn(straight) : "well the only one i know who captured any region's attention and any means everywhere is Mrs Melody Sanders Harrison and she did that from this very region(all laugh..lot) cause none touched the feat she did and none have been able to discern how she did what she did so no that hasn't been touched and that height won't be reached in quite a while now i guess(all laugh..lot)"
Press : "but you defeated..(all laugh..lot)"
Dawn(straight) : "firstly in actual terms her last selected compeer went to see the arrangement for next part of the conference beforehand and no more(all laugh..lot) then for the illusion no i didn't defeat her she did it herself completely unbeknownst to her and it was done the same way i did it after my first journey there's a limit for physique to take the pressure of psychic charges and both of us breached it the only one that succeeded from our cases certainly from my case if not my sister's are only Mr and Mrs Blackmore and they didn't make the mistake we did(all laugh..lot)"
Press : "but if you're referring to personal life then it's upto chance we can add from experiences compiled over millenias(all laugh..lot)"
Dawn(straight) : "well the whole point is that part of life but the thing is that unlike my sis i was groomed first hand i knew everything but my desires didn't hence the clash it has nothing to do with what you're referring to on my or her relationship status(all laugh..lot)..or our physical forms for that matter(all laugh..lot)"
Press : "well we're only trying to understand the middle part of your career Mrs Glenol the start was incomparable the end has been impossible(all laugh..lot)..and these two moments itself has fetched you 4 Majors with another at Kanto no matter who actually won that as you said(all laugh..lot)..and still in between that there's a runners up medal at Hoenn final against Mr Jerome Georges whose an all timer at GoC and he barely kept up with you in the initial rounds and that was a great match and again Mr Georges is in GoC by all means all times as well we understand the things you were going through emotionally till Ransei even but that Pasio journey of Ms Berlitz is what's really confusing to us Mrs Glenol(all laugh..lot) cause of the lows and highes right before and right after that seems too far fetched still to our cognition(all laugh..lot).."
Dawn(straight) : "well someone with a better preparation before start can do what i did a month before but you see it depends on the bunch and that's all and there too none will surpass my record on the number of gyms and contests and all so no the first was not incomparable it isn't surpassable there's a big difference mathematics would remind us the most by means of self sufficient failure in terms of axioms(all un..)..and the last one too possibility is a very subtle thing compeers it's not impossible again it's just not attainable(all burst..) and the reason is neither my bunch nor i had to do anything here at all it was our pleasure and privilege to be able to have so e good times with our plane agnostic compeers thus it's never attainment but entailment as equanimity isn't available to itself but only it's self again(all burst..).."
Press : "indeed we agree Mrs Glenol(all burst..) but in that way as you said you were gifted 4 major titles so that you can win this one your very last and still none knows if you're a all in one specialist or a co-ordinator or a general trainer Mrs Glenol except the so called Elusive🐉you've done everything(all burst..)..but does this also suggest that you are that itself(all un..,Dawn..Derelicts of Honesty,all un..).."
Dawn(laughing..) : "well if i really was that i wouldn't have bothered to enter Hoenn would i what could i get more than what I would've roaming around Sinnoh in wilderness none would've been able to object to that in the name of rangers and freelance services could they especially the last one as a profession after the whole chapter of 9 City Murmurs(all burst..,Melody..Logic's rigorous Coronation Organizer Association Bully🐉,all un..)..for the other two questions of yours well there's a reason why May says that she is a 3 time champ only as does Drew(all burst..,Gus..we shared the jouleasy too sweetie didn't i say first time I'm Drew hum sweetie hum,all un..,Gus..hum,Solidarity!Self-Lessness!Sacrifice!Commitment!,all un..,Guang..Educatio,all un..long,May un..long) and i do say the same thing i only have this Johto League to my credit compeers the rest Sinnoh Contest,League and Frontier belongs to my dear big bros and and ofcourse uncle crook(all laugh..lot,Dawn..That Feeling,all burst..)..then the next one as i said i didn't win it fully(all laugh..lot) so yeah i know what is what hence i tell you to keep off the poison of the brown emotion in the soothing comfort of green grass especially at Johto as my sis has been telling you for 20 years now about 8,8 in psychology of effusiveness(all burst..,Melody un..)..anyway about my Pasio journey well i wasn't in the right state of mind and didn't do the right things that's all then i met my other half in physique finally hence returned else i would've been gone"
Press : "well in accordance to that and what happened to Mrs Freesia..(all laugh..lot)"
Dawn(straight) : "has nothing to do with me or my compeers i know my bro and i know my sis and that's all..and about your questions in gross Mrs Freesia Waterflower Ketchum was absent from public for 3 months after 41 Years constantly spending under the spotlight and those who can't see the subsequent 2.5 will not see beyond her physical existence hence they should ask this question after she's gone may be tomorrow and no more can i say except that she wasn't involved in profession then her bunch wasn't there with her she was doing some nasty things but they only affected her and she stayed at a resort which was the property of Mr Kumaglak Kunilak whose her little bro who employed two dedicated staffs to look after her who were only Mr Brauch Xarar and Mrs Sarah Nomis Xarar who were among the foremost of people(all burst..,Guang and Gus un..) who knew her from way before she was born(all laugh..lot)..physically(all burst..)..they did what they could do none can stop the reality of Freesie Flower if that being decided to do something unless for her involution as you understand hence it was the best optimization indeed(all laugh..lot)..as why that being would want to do that it was only the testing of existense of the involution that she was trying and that's also now too minutely well documented in children's rhymes too i hear(all laugh..lot).."
Press : "perfectly comprehensive Mrs Glenol thanks(all burst..) but one thing is still unclear Mrs Glenol is that in case of Mrs Harrison she never was a travel companion of your mentors and the others who have been are in the gym leader's positions at various cities who travelled at Sinnoh at the time Mr Guang Ketchum did with his host Daily Hologram the most esoteric one(all laugh..lot)..so does it imply anything as to that they made sure you couldn't do it without them and in case of Mr and Mrs Blackmore they weren't what you were at first region and yeah Mrs Harrison did travel with Dr Oak throughout Pasio and Johto doesn't that help her results somehow (all laugh..lot)"
Dawn(straight) : "well the last i can't answer you when they started i was still in elementary(all laugh..lot) and i had not much interest as you all know about the second Mr and Mrs Blackmore picked what we did later and it was done through a much prolonged span of time hence they had more control over everything Mrs Blackmore bloomed late and when she did none came anywhere near her in terms of matching her except her other half who still only drew as you know..(all laugh a lot)..(all burst..,Drew un..)..anyway jokes aside(all laugh..lot,Gus..hum,Civility!,all un..,Drew and May too un..) Mr Blackmore caught a Flygon as his fourth ever catch he did have Mr Tony Thompson with him but then he wasn't at the level he was by the end of that journey he's telling that to all who'll listen for the last 18 long years now(all laugh..lot,Genkai and Blossom,Gladys,Gus..Tony Boy,all laugh..lot) and so Mr Blackmore had certainly what it takes to do what i did but he didn't want to do it then he did it himself.."
Press : "sorry for an interruption but did he somehow didn't do it in consistence to his personal life as you mentioned earlier about parity and rest(all burst..)"
Dawn(straight) : "most definitely there's no other reason cause he knew each of the things i knew infact anyone knows those cause they were being published everyday just as i finished which is the single most important reason why my personal life became extremely difficult subsequently(all laugh..lot) and about your first question whether they made sure of our lifelong faithfulness or not i just gave you the answer of that(all laugh..lot)"
Press : "well you've directly stated and Mrs Harrison implied at her Unova journey and later at Rimsable that you two weren't treated well singularly by Mr Guang and Gus Ketchum and specifically by the latter(all laugh..lot,Iris,Serena,Dawn and May..TYRAniTAR,all burst..)..and only all of their si..(all burst..).."
Dawn(straight) : "yeah i get i get it(all laugh..lot) well that's cause the reflections of other shore of emotion aren't available to public and it shouldn't either all puns intended my hubby would add and especially Mr Harrison you may look at his daily reporting to 9 City Conglomerate for 15 years from Rimsable if you're still not clear(all burst..,Melody un..)..and you know about the works and influences of my dad Mr Glenol(all laugh..lot).. anyway back to the privilege or opportunity to be a travel companion of theirs how would you categorize my case(all laugh..lot)"
Press : "well your case was an opportunity as was Mr and Mrs Blackmore's unlike say Mr Salusar and Mrs Rarcia(all laugh..lot)..then Mrs Harrison did travel finally with you all at Sinnoh but you were refused at 2 whole regions the second of which is very difficult to comprehend from our part on their actions throughout(all laugh..lot)"
Dawn(straight) : "all of our cases were opportunities and there's a trend in recent times to take Mrs Rarcia for granted i'm sure we all know her full career right especially the Kanto League final and the Elite4 challenge(all laugh..lot).."
Press : "there's no doubt about that Mrs Glenol but the thing is that you didn't have the privilege she had during undercover ok that's accident but still she was accepted to actually go to the spot and then Kanto and all being properly guided(all laugh..lot)
Dawn(straight) : "that's because of her career before she was a 7 time champ in first 7 regions all by herself if i had that behind me i would've jumped into undercover too may be cause atleast by means of battling say even 5 on 1 even with boosters but fair rules and no more than 5 trainers each having own pokemon you can't win fair battles atleast again me even if don't lose(all laugh..lot)..and definitely not get away from that without further consequences if my bunch has seen Unova 0th hand having come from exact same valleys like on a contract(all laugh..long)..except for Mr Nofcan none would've beaten her at Kanto and he did it in a 16 minute very last round case that too his Goodra stood up at 18 Mrs Rarcia's took 6 exact more seconds and no more(all laugh..lot)..then at Elite4 challenge she lost only so few rounds no matches barring Mr Marc Litman reasoning unnecessary after the very nane(all burst..,..Marc The Mr Litman)..and well about the main difference between she being pampered and me being rejected is all cause what i understood much later that i wouldn't be alive physically if i was allowed which no matter my wish would've had repercussions in society who don't understand all the intricate nuances of Pokemon Training including researchers and all included the things i've come to know in this journey even Dr Oak i believe didn't know before the Unovan Trouble started 2 years back infact none knew among us now it's public knowledge without the counter which select few can and will know or be given which has been in my case this time(all laugh..lot)..it's tacit knowledge and one has to face say a Heatmore at a volcano bed to really get it(all laugh..lot) but the thing is that it's given to me cause i was told about these there itself by the two persons you mentioned they subsequently only validated themselves at Unova cause that wasn't for learning it was for application it's like death none will listen to our practice or sentiment or life stories there or the fact that i want to look at life once more it's like that the spontaneity there's no knowledge only application which isn't the case if seen closely but that's in terms of self in terms of it's about training and meaning as they've been saying for 41 years now that too constant(all laugh..lot)..so as for that matter no i only understood later their actions as did their own other halves and Mrs Fressia Waterflower Ketchum was traveling throughout Unova and Kalos and I don't need to talk about her so if it was a mystery to her what her other half was doing most definitely it will be to his little sisters(all laugh..lot)..so yeah this lifelong faithfulness benefits people neither the objects nor their practice unless for the later they acknowledge what they want and what they're at present doing so yeah you can say in a way they only made sure i know that i wad their very own much more than spatiotemporal scales can ever define which if certainly i rememberd i would've had an easier journey less wasteful that's all but it all is conjecture now and hence implausible to talk about(all laugh..lot) and as for Melody well her case was much more difficult and different to me but again she admits this all time she should've done what i should've too that is travel with any or all of the people that we were suggested to travel with exactly what Mrs Rarcia did after undercover that or talk with the suggested people among whom also are the exact persons that we both eventually will talk with all our lives as a signed agreement by your consensus law and nothing else(all laugh..lot)"
Press : "well about that Mrs Glenol,Mrs Harrison's case we all know your case wasn't so after Hoenn so it must be extremely dangerous what you were exposed to in the name of practices at Sinnoh(all burst..)"
Dawn(straight) : "well you said the word yourselves (all burst..,Sirichai.. that's a DANGEROUS consequence Sirich retrospectively exactly how Ciddu had to leave his job you know,all un..,..The Eureka Ones,all un..) i turned 20 and found myself alone suddenly with confusions all over so i was talking with people whom i only should've spoken with to use lexicon properly(all laugh..lot) and yes it was dangerous in terms of training it always is dangerous to keep a Spinda even as you know Dr Serena Toganne Tomkins' stories of her co-ordinator days at Unova the two months(all laugh..lot,Serena bursts..).."
Press : "yes we acknowledge that but also we hear how she too the very next month after being dragged all over the whole Hoenn and Ransei region for a month got married the very next month so she too was helped in a way only after she attempted to face a not fully tamed Kommo-O with a domesticated Spinda in a contest second round firstly is that the test of courage secondly is that a pattern in this(all burst..,Guang,Gus,Bazugh and Serena un..).."
Dawn(straight) : "no it's not a test of courage it was an act of emergency to preempt her from the mythical Unovan bushes where none goes first up pro or amatuer with even a Marill(all burst..,Guang and Gus..Mr Skechit why you think and do and not do or think so after thinking or doing/thus jumping into ocean with Marill..will be ruing,all un..)..and she wasn't alone but had about 100 of her siblings looking after her day and night infact her other half was the main field referee in that occasion itself by procedural decisions(all laugh..lot) so only there it might've been an act of courage(all burst..,Bazugh and Serena un..)..she couldn't do it alone hence was left not lone as they'd say(all laugh..lot) but we could and we were champs hence our cases were handled differently Mrs Harrison not going to Hoenn i know for record first hand was a pure case of inter sibling rivalry with our big bro Mr Gus Ketchum taking the side of Dr Oak for a change(all laugh..lot,Melody bursts..)..besides my nephews are still unsure if i'm elder to her or the reverse(all laugh..lot,Melody too laughs..lot)..then with this precursor the pattern is familiar that if i disgorge a food you can't make me eat that that was the case in both our cases she did it unknowingly in it's entirety and that was proven by her Frontier journey and the 4 journeys she had her family with her she reached the last step one loss against Dr Oak is considered a professional debut to training those who faced him know that like Mrs Riley Rednak Reid has been saying for 27 years constant yeah almost half her life(all laugh..lot)..physical that is(all burst..),and then she did the unthinkable here exactly one month short of 20 years back beating Colony Monogram and Column Jargon leaving the ball to Unovan courts absolutely as she postponed it for 9 years and most of all saved Mrs Blackmore from University classes on psychology(all laugh..lot,May bursts..)..then she did do that final with me where i didn't know one damn thing as to why i was doing what i was doing except for the fact that i knew that i was doing what was being asked of me to be done by rationality not logic(all burst..)..so there's no credit for me in there and the fact of the matter is i didn't need to go against her i could've done it the month later and the Frontier earlier but that means i would've faced Mr Genkai Oak rest insured by Mrs Harrison herself(all burst..,Melody un..)..she never wanted to win unlike what i did and all except Kalos and Unova where she failed to get past Round 32 and i didn't finish even a year in 3 successive regions two of which were Kalos and Unova that's for stats(all laugh..lot)..those two are the hardest regions in terms of the trekking Unova is legendary for the Elusive bushes Kalos for it's unofficial challenges with say a Zygarde against a Magnezone that's inhuman as Dr Jackal noticed still a physical teenager(all un..,Dawn..Tickle Mastery,all un..)..anyway the way i was taught or Mrs Harrison understood things we needed the daily personal care which we ignored completely cause we were so emotionally attache to our bros we didn't see past that and then had our own incentives Mrs Blackmore was far more infatuated with bro than both me and Melody combined but you see she is a bit shrew thus only there's 3 exact leagues of her own rest 3 she drew(all burst..,Gus..Culture,all un..,Drew and May un..)..so the pattern is like that and yeah all of us sisters were too emotionally attached to bro,Bonnie had to talk with Gungie each day and never once with her hubby on how to educate her daughter infact her hubby was banished from home as a revenge for not allowing her to travel with her big bros no matter how old Bambie was with them anything is possible(all burst..,Guang..Hum,Faith!Trust!Sercurity!Dependability!Insurance!,all un..,Drew and May un..)..and the reason is best explained by their respective flowers only and their influences on the likes of Mr Redzep Rafolrat never elaborated further(all burst..,Guang and Gus..Mr Hummm-Berto RUOUGH-ON-STAT..,all un..long,Drew..Culture..,all un..,Guang and Gus un..long) hence during their breakup years most of their sisters and brothers were also against each other at home only making it clear to them that it made no sense at all(all laugh..lot)..and the reasoning behind is that the inter brothers and inter sisters relationships with them isn't very proper to elaborate in public i know the best i was 18,Lillie flower was 36 not months physically(all burst..),and Freesie was 33 and we argued each day atleast an hour on hour itself(all laugh..lot,Bonnie,Serena,Melody,Iris,Dawn and May..Daily Daily WaterFire/Hasty Hasty Logic mire,all un..,Guang and Gus un..)..so yeah that's the pattern and if we do our job properly it's going to repeat again next gen(all laugh..lot).."
Press : "thanks for the detailed clarification Mrs Glenol(all burst..) but from your surmise it seems that Dr Glenol was sent to Flexdijon in that turmoil with an attempt of heist there by the runaway Unova Purge Group so if that's their relationships with their brothers then it doesn't seem to always follow rationality to us(all un..,Sirichai..ofcourse i know what to do if Daphie is ill what do you mean huh,the distinction of pollution compeer..shh..to sense faculties no more seem, countering ain't it rhyming with how could it would it be if or not if if application is added of duh,all un..long,Sirichai..The Lessened Feather,all un..long)
Dawn(smile) : "well our brothers always take it upon them if you notice properly that was what it was all about and no his family knew it and he was safe too physically no more than a lonh flight risk as they ever said that too we know(all laugh,Sirichai,Melody,Drew and May..Too Sweetie,Dawn laughs..lot saying,atleast we didn't say in public like born as separate only to forget you're not being what i'm right now,all laugh..lot,Gus,Guang,Melody,Drew and May..cunning,all laugh..lot,..So Sweetie,Dawn bursts..)"
Press : "we see thanks for clarifying again Mrs Glenol(all laugh..lot)..but now what we want to ask you is that how's your relationship with Mr Jerome Georges(all laugh..lot)"
Dawn(smile) : "as good as it gets"
Press : "well he recently said that he had long past the sentiment that you were only presented as such by certain substances and hence afterwards your life was in shambles now he says jokingly that he was so glad that you weren't ok mentally on the day of the Hoenn Contest final(all laugh..lot)..but what we want to say is that Mrs Glenol that in the last 8 months all the acrinonies surrounding you and your mentors and all seemes to have finally dissapread for good for the rest of next generation atleast and we hold this factor down to the in their words eloquence of the Riumina herd there at Oak Labs HQ(all laugh..lot) do you and your other half from first hand believe it or just the fact finally all the Elite4 facilitators has had to acknowledge their works finally after their Hands-On poof of 8,8(all laugh..lot)"
Dawn(smile) : "well for both of us like all of you too i know it's just that they've been debarred from leaving physically through a concensus among the smogons and that's all"
Press : "indeed we completely agree with both of you Mrs Glenon that they managed to put their petty differences aside but on a lighter note did you notice at anytime that you were forwarded as 8,8,8,8 so how would you validate that(all burst..)"
Dawn(laugh) : "well i did get do what can't be surpassed so that's for your 4 8s and because still there's a month extra that i needed to get started means it's not 9,8,8,8 that's all i can say(all laugh..lot)..(all burst..)"
Press : "thanks Mrs Glenol for clarification(all laugh..lot) and well one of the things is that you and Mrs Blackmore were unique in terms of being their travel companions rest either had experience or were traveling alone like you two so comparing to her how do you rate yourself(all laugh..lot)"
Dawn(laugh) : "well she's in gallery of communications in profession(all burst..,Gus..SHUT UP YOU BRAT!BOTH of have EDUCATION on how to approach for a discussion for a proper date selection for the signature UNLIKE your sudden appearance like an uncivilized shrew,yeah reflectively, atleast both of us went prepared unlike your one hour in the middle take that you brat,all un..,Drew and May un..) and in person i had a fight with her other half before coming here as he always told me to be calculated in press which i haven't been cause he refuses to train here neither me nor his wife nor anyone else did and that's all that's needed to produce a result for a change(all burst..Dawn,Drew and May un..)"
Press : "well we all wish for the best of joy for all of you concerd fervantly(all laugh..lot) well another thing is that Mrs Glenol is that how's your relationship with Mrs Waterflower Skechit cause after Sinnoh neither you nor she ever spoke with each other(all burst..,..DaisyDevil)"
Dawn(laugh) : "the right word should've been communicated burst..,..DaisyDevil) and well again it's just that she could be accused of conflict of interests and i don't have to say more to enter her office room anyway The Waterflower Sisters are like that compeers if you don't recognize their name you don't get anything at all about them the eldest the most in the reverse way(all laugh..lot,..DaisyDevil)"
Press : "thanks and again our fervent wishes(all burst..) lastly then are you going to work as a judge or ranger(all laugh..lot)"
Dawn(laughing..) : "well it's so fitting compeers that my travelling career ends like this with this note cause i started with this question of actions and habits and practice of self unbinding to Simile Popcorn who as you mentioned the word referred me to Lastly Phantom and thus i began he'd say to elaborate not actuate..(all burst..)..(all un..) so i'll end with this too what i do next look tense says Phantom..anything else countering?(all burst..)..(all un..,Iris and Dawn..Lastly The Playboy Confirm,all un..)"
